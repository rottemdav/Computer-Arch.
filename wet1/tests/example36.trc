8 7 3 2 local_history local_tables not_using_share
0x14bc T 0x8ec3285c
0x1d60200 T 0xe9ab40
0x14bc T 0x8ec3285c
0x92bc N 0xf6455358
0x1d60200 T 0xe9ab40
0x1d60200 T 0xe9ab40
0x2402c N 0xbfcb66cc
0x1d60200 T 0xe9ab40
0x1d60200 T 0xe9ab40
0xd14 T 0x9138ad28
0x2402c T 0xbfcb66cc
0x1d60200 T 0xe9ab40
0xd14 T 0x9138ad28
0xe8dec T 0x90c8296c
0x14bc N 0x8ec3285c
0x2402c N 0xbfcb66cc
0x92bc T 0xf6455358
0x1d60200 T 0xe9ab40
0xfb78c4 N 0x5bcf7a0
0x42ffc204 T 0xdceff4
0x74f74 N 0xc34f8