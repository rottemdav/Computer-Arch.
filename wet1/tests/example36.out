0x14bc N 0x14c0
0x1d60200 N 0x1d60204
0x14bc T 0x8ec3285c
0x92bc T 0x8ec3285c
0x1d60200 T 0xe9ab40
0x1d60200 T 0xe9ab40
0x2402c N 0x24030
0x1d60200 T 0xe9ab40
0x1d60200 T 0xe9ab40
0xd14 N 0xd18
0x2402c N 0x24030
0x1d60200 T 0xe9ab40
0xd14 T 0x9138ad28
0xe8dec N 0xe8df0
0x14bc T 0xf6455358
0x2402c N 0x24030
0x92bc T 0x8ec3285c
0x1d60200 T 0xe9ab40
0xfb78c4 N 0xfb78c8
0x42ffc204 N 0x42ffc208
0x74f74 N 0x74f78
flush_num: 9, br_num: 21, size: 2376b
